features/pickle/convert-info.feature

```diff
@@ -62,7 +62,6 @@ Feature: convert package.xml to information files
       {
           "name": "dummy",
           "type": "extension",
-          "version": "3.1.15-beta",
           "description": "This is a dummy package",
```

Comment:
I am not sure why it is required to remove this? It is actually good to test this part.

Now tests pass btw, do you still think we should remove that part? And if yes, what would be the reason?

Reply:
I honestly have no idea anymore. I think I just updated these to make tests pass, so you can ignore this change if all is good on master.